packages/pacget-git/pacget-git.pacscript

```diff
@@ -0,0 +1,44 @@
+#!/bin/env bash
+#-*- mode: sh ;mode:lsp;-*-
+name="pacget-git"
+pkgname="pacget"
+pkgdir="/usr/src/pacstall/${name}"
+
+maintainer="Erik Lundstedt <erik@lundstedt.it>"
+url="https://gitlab.com/Erik.Lundstedt/pacget.git"
+
+license="Don't Be a Jerk"
+depends="fzf curl bash"
+gives="$pkgname"
+description="A fully foss, lightweight fuzzysearching TUI frontend for pacstall using fzf"
+removescript="yes"
+
+debian=TRUE
+ubuntu=TRUE
```

Comment:
```suggestion
debian=TRUE
ubuntu=TRUE
```
Not needed!

Reply:
i think its nice to have
although best case scenario would be if something similar got implemented directly into pacstall

Reply:
okay.